manifests/runtime-config/bosh-dns.yml

```diff
@@ -35,3 +35,62 @@ addons:
             ca: ((dns_healthcheck_tls_ca.certificate))((dns_healthcheck_tls_ca_old.certificate))
             certificate: ((dns_healthcheck_server_tls.certificate))
             private_key: ((dns_healthcheck_server_tls.private_key))
+
+variables:
+- name: dns_healthcheck_tls_ca
+  type: certificate
+  options:
+    is_ca: true
+    common_name: dns-healthcheck-tls-ca
+
+- name: dns_healthcheck_server_tls
+  type: certificate
+  options:
+    ca: dns_healthcheck_tls_ca
+    common_name: health.bosh-dns
+    extended_key_usage:
+    - server_auth
+
+- name: dns_healthcheck_client_tls
+  type: certificate
+  options:
+    ca: dns_healthcheck_tls_ca
+    common_name: health.bosh-dns
+    extended_key_usage:
+    - client_auth
+
+- name: dns_api_tls_ca
+  type: certificate
+  options:
+    is_ca: true
+    common_name: dns-api-tls-ca
+
+- name: dns_api_server_tls
+  type: certificate
+  options:
+    ca: dns_api_tls_ca
+    common_name: api.bosh-dns
+    extended_key_usage:
+    - server_auth
+
+- name: dns_api_client_tls
+  type: certificate
+  options:
+    ca: dns_api_tls_ca
+    common_name: api.bosh-dns
+    extended_key_usage:
+    - client_auth
+
+## Our additions
+
+- name: dns_healthcheck_tls_ca_old
```

Comment:
BTW, I am not able to find where we implement the certificate rotation for this project. We do in paas-cf, but not sure if we do here. 

Should we include these variables if they are not useful?

Reply:
These vars are needed to satisfy the static analysis tests in Travis, which insist on all `*ca*` things having an `_old_` variant concatenated.  We could get by with dummy values in there, not backed by bosh-generated "proper" CAs, but it seemed like more work to do that than simply to include them as carbon copies of their non-`_old` siblings.

Reply:
Oh, do we have such test in this project?

Reply:
I really thought we did, but it's very possible we'd assumed such a test existed, based on paas-cf.

Given the `_old` vars are in production right now, I don't think the overhead of having them there is significant - it would be more effort to remove them.

I don't feel this particularly /strongly/, however :-) @alext @richardTowers @henrytk opinions?

Reply:
IMO I would delete the `_old` if it is not being tested and the feature to rotate does not exist. It is misleading and it does not provide anything.

Also it is easy to remove and to add later if needed.

Reply:
I think let's remove them.

Reply:
@richardTowers @keymon Done. Please re-review.

Reply:
LGTM. I am rerunning in my env and would merge soon.